shared-libs/phone-number/src/phone-number.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-empty */
```

Comment:
Again, we should probably be documenting each one instead of just turning this off

Reply:
Definitely. I think the fix for this one is just to put comments in the empty catch blocks. Maybe do that instead of disabling the rule altogether.